### ISSUE TITLE
Failed to generate Mesh

### ISSUE THREAD

**Issue by seulqxq** [Anttwo/Frosting#8]
Hello, I made an error after training, and the Mesh file was not generated in the output. Do you know the reason?
![image](https://github.com/user-attachments/assets/ae102058-1229-437c-a3f3-0fa2b40481f1)

**Comment by zb8c5ek** [comment]
indeed, i am facing the same problem here ... and i am using a docker image container.

**Comment by seulqxq** [comment]
> indeed, i am facing the same problem here ... and i am using a docker image container.

I remove the library nvdiffrast, which conflicts with my environment. You can try it, and hope it will be useful to you.

**Comment by zb8c5ek** [comment]
> > indeed, i am facing the same problem here ... and i am using a docker image container.
> 
> I remove the library nvdiffrast, which conflicts with my environment. You can try it, and hope it will be useful to you.

thx ! though not complaining about the env, after uninstalled the nvdiffrast, it can run further now.
